compiler-core/src/parse.rs

```diff
@@ -678,6 +683,35 @@ where
         Ok(Some(expr))
     }
 
+    // A `use` expression
+    // use <- function
+    // use <- function()
+    // use <- function(a, b)
+    // use <- module.function(a, b)
+    // use a, b, c <- function(a, b)
+    // use a, b, c, <- function(a, b)
```

Comment:
Can it also be used in pattern matching like let:

> let Cat(name: name, cuteness: _, age: _) = cat

Reply:
Nope, at least not with this first version.